Makefile

```diff
@@ -39,7 +39,7 @@ export GO111MODULE=on
 #
 # Kubebuilder.
 #
-export KUBEBUILDER_ENVTEST_KUBERNETES_VERSION ?= 1.23.3
+export KUBEBUILDER_ENVTEST_KUBERNETES_VERSION ?= 1.24.1
```

Comment:
In theory this is fine, in practice it will only work on Linux > https://github.com/kubernetes-sigs/kubebuilder/pull/2709#issuecomment-1148492299